src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+import { PersonRow } from '../PersonRow';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { useParams } from 'react-router-dom';
+
+export const PeopleTable = () => {
+  const { slug } = useParams();
+  const normalizedSlug = slug || null;
+
+  const [loading, setLoading] = useState(true);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig